contracts/src/v0.8/vrf/dev/VRFCoordinatorV2_5.sol

```diff
@@ -27,9 +27,11 @@ contract VRFCoordinatorV2_5 is VRF, SubscriptionAPI, IVRFCoordinatorV2Plus {
   error InvalidRequestConfirmations(uint16 have, uint16 min, uint16 max);
   error GasLimitTooBig(uint32 have, uint32 want);
   error NumWordsTooBig(uint32 have, uint32 want);
+  error MsgDataTooBig(uint256 have, uint32 max);
```

Comment:
This error is also defined in the other PR

Reply:
I based this PR on the previous PR :)
Anyways, we can merge only one by one right